main.py

```diff
@@ -270,7 +279,10 @@ def get_geometry(df, year):
     # Gets the TIGER geometry for the census tracts
     geom = tracts(state="TX", year=year, cache=True)
     geom = geom[["GEOID", "geometry"]]
-    df = df[["GEO_ID", "NAME", "indexed_vulnerability", "eaz_type"]]
+    output_cols = ["GEO_ID", "NAME", "indexed_vulnerability", "eaz_type"] + [
+        w["column"] for w in WEIGHTS
+    ]
+    df = df[output_cols]
```

Comment:
everything else is just formatting. I've added the weight columns to the output geojson